styles.css

```diff
@@ -226,36 +231,40 @@ nav {
   line-height: 24px;
   size: 16px;
   text-align: left;
+  color: #091e42;
+  padding-bottom: 10px;
 }
 
 .tech-skills {
   display: flex;
   flex-direction: row;
-  gap: 10%;
+  gap: 8%;
   height: 40px;
   width: 70%;
   align-items: center;
 }
 
 .skillset {
-  border: 1px solid #8993a4;
+  border: 1px 'solid' #8993a4;
+  color: #091e42;
```

Comment:
- [ ] To make sure you are following the design guidelines defined by the template please include borders around your project technologies (e.g., "JavasScript", "HTML" and "CSS"). Also please use the font-family `Inter` for the project technologies list as in the figma design they are using font-family `Inter` as well.

|Figma design| Your design|
|-------------|-------------|
|<img width="316" alt="Screenshot 2023-05-06 at 2 11 45 PM" src="https://user-images.githubusercontent.com/44798044/236614974-351dc277-bfb7-4dc5-832d-2962f3a77a08.png">|<img width="360" alt="Screenshot 2023-05-06 at 2 00 59 PM" src="https://user-images.githubusercontent.com/44798044/236614991-703649de-da74-4b1a-8098-da172a9e6dcb.png">|